src/components/Checkbox/Checkbox.js

```diff
@@ -126,6 +126,8 @@ Checkbox.defaultProps = {
   indeterminate: false,
 };
 
-export default React.forwardRef((props, ref) => (
-  <Checkbox {...props} forwardRef={ref} />
-));
+export default React.forwardRef(
+  Object.assign((props, ref) => <Checkbox {...props} forwardRef={ref} />, {
+    displayName: 'Checkbox',
```

Comment:
this displayName may be problematic, because from my understanding this means component tree will end up looking like
```
<Checkbox>
  <Checkbox />
<Checkbox>
```
this could break a lot of unit tests that do something like Wrapper.find('Checkbox') because suddenly there are two checkboxes, instead of one

it's better to have something like
<withForwardedRef(Checkbox)>
 <Checkbox>
</withForwardedRef(Checkbox)>